webpage/make_index.py

```diff
@@ -49,7 +49,8 @@ def extract_thumbnail_and_title(nb_path):
         index.write(front_matter)
         index.write('<div id="gallery">\n')
 
-        for fname in sorted(glob.glob(os.path.join(os.path.pardir, 'notebooks', '*.ipynb'))):
+        notebooks = glob.glob(os.path.join(os.path.pardir, 'notebooks', '*.ipynb'))
+        for fname in sorted(notebooks, reverse=True):
```

Comment:
To get the last notebook first in the gallery all we need was to reverse the order of the notebook list.